### PR TITLE
[chrome] Drop copy-pasted duplicate "types" from JSDoc

### PR DESCRIPTION
**Pull request by fregante**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

---

These JSDoc comments were copy-pasted from Chrome’s documentation and duplicate our own types (sometimes giving conflicting information). They're lengthy and don't add anything useful, so this PR drops them all

Related:

- https://github.com/DefinitelyTyped/DefinitelyTyped/pull/64272

### CODE REVIEW

File: types/chrome/index.d.ts [hunk 1]
```diff
@@ -1432,26 +1330,22 @@ declare namespace chrome.contentSettings {
...
-         * function(array of ResourceIdentifier resourceIdentifiers) {...};
-         * Parameter resourceIdentifiers: A list of resource identifiers for this content type, or undefined if this content type does not use resource identifiers.
-         */
-        getResourceIdentifiers(callback: (resourceIdentifiers?: ResourceIdentifier[]) => void): void;
+        getResourceIdentifiers(
+            callback: (
+                /**
+                 * A list of resource identifiers for this content type, or undefined if this content type does not use resource identifiers.
+                 */
+                resourceIdentifiers?: ResourceIdentifier[]
```
[the remark this answers is not in the document]
Comment (matcarlson) [reply types/chrome/index.d.ts:1344]: IMO yes, this is much easier to parse in IDEs like this, and it's not any worse at the least in code. You can also use the inline form when possible, e.g.  `/** The reason for this variable */ varName: string[],` for shorter sentences / where it looks good.

File: types/chrome/index.d.ts [hunk 2]
```diff
@@ -3489,31 +3314,29 @@ declare namespace chrome.events {
...
          * Registers rules to handle events.
          * @param rules Rules to be registered. These do not replace previously registered rules.
          * @param callback Called with registered rules.
-         * If you specify the callback parameter, it should be a function that looks like this:
-         * function(array of Rule rules) {...};
-         * Parameter rules: Rules that were registered, the optional parameters are filled with values.
          */
+        addRules(
+            rules: Rule[],
+            callback: (
```
Comment (matcarlson) [inline types/chrome/index.d.ts:3331]: This appears to be a duplicate of the overlaod at line 3336. Prefer to just leave the callback parameter as optional rather than an overload that makes it optional. 

tl;dr> Can you make this callback optional, then remove the overload below?

**Comment by matcarlson** [comment]
Suggested edit:
```diff
diff --git a/types/chrome/index.d.ts b/types/chrome/index.d.ts
index 49544cda56..d467c50373 100644
--- a/types/chrome/index.d.ts
+++ b/types/chrome/index.d.ts
@@ -5962,7 +5962,7 @@ declare namespace chrome.permissions {
     }
 
     export interface PermissionsAddedEvent {
-        addListener(callback: (/** The newly-acquired permissions*/permissions: Permissions) => void): void;
+        addListener(callback: (/** The newly-acquired permissions.*/ permissions: Permissions) => void): void;
     }
 
     /**

```

**Review by matcarlson — CHANGES_REQUESTED** [review]
Overall looks great. I suggested one very minor change, and one that you should remove the original definition.

File: types/chrome/index.d.ts (hunk 2 above)
Comment (fregante) [reply types/chrome/index.d.ts]: Good find, dropped

**Comment by fregante** [comment]
Thanks for the review! Changes made.

**Review by matcarlson — APPROVED** [review]

**Comment by fregante** [comment]
This only changes comments, no tests/behavior is changed

**Review by matcarlson — APPROVED** [review]

**Comment by fregante** [comment]
Ready to merge
